### ISSUE TITLE
Not sure if set.seed function needs to be added again for model tuning/cross-validation

### ISSUE THREAD

**Issue by acholj** [acholj/Updated-DSCI-Group-Project#1]
The best K value changes every time I run all cells, but i remember reading in the textbook to only set.seed once so I'm not sure if I should add another one or not. I haven't but if you know that answer, feel free to make a change if needed.

**Comment by CeciH1026** [comment]
True!!!! It is probably because of the system of Jupyter that we need to code by chunks?
